test/microstrain_test.hpp

```diff
@@ -4,19 +4,28 @@
 	The goal is to provide a standardized interface for testing, regardless of the
 	backend framework being used.
 
+	Currently, only EXPECT interfaces are implemented for custom assertions. They
+    will be added as needed and when they can be fully tested.
+
 	TODO: This should be moved to its own repository so it can be used by multiple
           projects
+
+	TODO: Find a way to reuse custom logic between WARN, EXPECT, and ASSERT interfaces
 */
 #pragma once
 
-#ifdef USE_DOCTEST
+#ifdef MICROSTRAIN_TEST_USE_DOCTEST
+#include <cstddef>
+
 #include <doctest/doctest.h>
 
 namespace detail
 {
-    void warn_c_strings_equal(const char* string1, const char* string2);
-    void check_c_strings_equal(const char* string1, const char* string2);
-    void require_c_strings_equal(const char* string1, const char* string2);
+    static constexpr size_t MAX_STRING_LENGTH = 1024;
+
```

Comment:
This needs to be used in the tests, i.e. `buffer[MAX_STRING_LENGTH]` instead of `buffer[1024]`.
I'd also recommend TEST in the name, so it's clear that it's not something to do with the code being tested. Maybe `TEST_CHAR_BUFFER_LENGTH`?